README.md

```diff
@@ -135,7 +140,21 @@ To run the sandbox unit tests run the following command:
 make sandbox-test
```

Comment:
Not in this PR changes but when I run this command I get this error:

make: *** No rule to make target `sandbox-test'.  Stop.